layers/+lang/sql/packages.el

```diff
@@ -117,3 +122,19 @@
 (defun sql/init-sql-indent ()
   (use-package sql-indent
     :defer t))
+
+(defun sql/init-sqlup-mode ()
+  (use-package sqlup-mode
+    :defer t
+    :init
+    (progn
+      (add-hook 'sql-mode-hook 'sqlup-mode)
+      (when (and sql-capitalize-keywords (not sql-capitalize-keywords-disable-interactive))
```

Comment:
Since you're using `:toggle sql-capitalize-keywords`, first test is redundant. You can reduce the expression to `(unless sql-capitalize-keywords-disable-interactive ...)`